.gitignore

```diff
@@ -10,5 +10,8 @@
 /lab2d/
 
 # Ignore saved models.
-/examples/results/
 /meltingpot/assets/
+
+# Ignore examples output.
+examples/rllib/.results/
```

Comment:
Sorry, did you say you were going to move this out of this change? I'm not clear that this path has changed here.

Reply:
Sorry done - I'll add this to the a3c example PR